hazelcast/src/test/java/com/hazelcast/query/impl/extractor/predicates/CollectionDataStructure.java

```diff
@@ -64,12 +63,12 @@ public boolean equals(Object o) {
                 return false;
             }
             final Limb other = (Limb) o;
-            return ObjectTestUtils.equals(this.name, other.name) && ObjectTestUtils.equals(this.power, other.power);
+            return Objects.equals(this.name, other.name) && Objects.equals(this.power, other.power);
         }
 
         @Override
         public int hashCode() {
-            return ObjectTestUtils.hash(name, power);
+            return Objects.hashCode(new Object[]{name, power});
```

Comment:
You should be using `Objects.hash` instead

Reply:
rev : f7f7c25e52bdd3a91e85a3c19a7264511b1abcab